src/components/dialog/DialogBase.js

```diff
@@ -70,6 +75,39 @@ export const DialogBase = ({
   return createPortal(dialog, document.body);
 };
 
+const makeDraggable = () => {
+  let x = 0;
+  let y = 0;
+
+  const ele = document.getElementById('dragMe');
```

Comment:
ref should be used here, which means we should probably also create a hook for it instead (similar to the package), but I think I prefer maintaining it here than inside of that package, especially since it's not a very "established" package

Reply:
yup, fair enough, thought the package was neat because you can set a drag target and handle ref, which can mimic the existing implementation of "handling drag bij clicking only the icon, but targeting the full popup".

Reply:
Doesn't seem super hard to implement ourselves either

Reply:
If its easy to implement it && that package in other draft is not reliable, then I guess we should go with our implementation